### ISSUE TITLE
Radarr exporter uses too many resources

### ISSUE THREAD

**Issue by ngosang** [onedr0p/exportarr#87]
# Details

Radarr exporter uses too many resources compared to Sonarr and Lidarr exporters. My movie collection is not that big. Maybe you can take a look at the endpoints you are calling and optimize something.

* Radarr => 739 Movies
* Sonarr => 132 Series 6450 Episodes

exportarr: v1.1.0
ENABLE_ADDITIONAL_METRICS: false
ENABLE_UNKNOWN_QUEUE_ITEMS: false

My fix for now is to configure a huge scrape_interval in Prometheus.
```yaml
  - job_name: 'radarr-exporter'
    scrape_interval: 600s
    static_configs:
      - targets: ['192.168.1.191:9708']
```

![image](https://user-images.githubusercontent.com/10577978/190276292-8f81886c-2b08-4c79-8052-0c36b44e376d.png)

Update:
After I changed the scrape_interval, I can see Radarr is using less resources too:
![image](https://user-images.githubusercontent.com/10577978/190280715-55774e15-5834-401b-8969-9fae8df6b06b.png)

**Comment by onedr0p** [comment]
Honestly I would say 5 minutes is the minimum interval that should be used for collecting metrics due to the nature of how exportarr works by making a ton of API calls.

**Comment by rtrox** [comment]
Looked at this a bit today. I don't think there's much that can be done about memory usage, Exportarr only makes a single API request for Radarr (to the `/movies` endpoint), the resource usage is just that the response from that endpoint is massive (my collection is much smaller than yours, and it's still over 400Kb uncompressed).

I did a little napkin math, one movie in the JSON response is roughly 7.8K uncompressed, so for 720 movies, the response is close to 5MB. So I'm not super surprised to see the memory usage is that high.

Network Usage, though, I think can probably be better. I played with the *arr APIs a bit, and they all support `gzip` compressed responses, but default to sending them uncompressed. I'm playing with a PR now to ask for GZIP compression in exportarr.
